api/src/main/java/com/messagebird/MessageBirdClient.java

```diff
@@ -950,7 +936,7 @@ public ConversationList listConversations() throws UnauthorizedException, Genera
      */
     public Conversation startConversation(ConversationStartRequest request)
             throws UnauthorizedException, GeneralException {
-        String url = String.format("%s%s/start", this.conversationsBaseUrl, CONVERSATION_PATH);
```

Comment:
you should update also this.conversationsBaseUrl with CONVERSATIONS_BASE_URL on line 951

Reply:
Updated, thanks 👍